spec/models/miq_report_result_spec.rb

```diff
@@ -37,4 +37,42 @@
       report_result.report_results.table.data.should_not be_nil
     end
   end
+
+  describe "serializing and deserializing report results" do
```

Comment:
@gtanzillo any way we can test PDF generation here also without getting a `NotImplementedError`?